src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,64 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String model;
+    private String color;
+    private int maxSpeed;
+    private int mileage;
+    private Engine engine;
 
+    public Car(String model, String color, int maxSpeed, int mileage, Engine engine) {
+        this.model = model;
+        this.color = color;
+        this.maxSpeed = maxSpeed;
+        this.mileage = mileage;
+        this.engine = engine;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getMaxSpeed() {
+        return maxSpeed;
+    }
+
+    public void setMaxSpeed(int maxSpeed) {
+        this.maxSpeed = maxSpeed;
+    }
+
+    public int getMileage() {
+        return mileage;
+    }
+
+    public void setMileage(int mileage) {
+        this.mileage = mileage;
+    }
+
+    public Engine getEngine() {
+        return engine;
+    }
+
+    public void setEngine(Engine engine) {
+        this.engine = engine;
+    }
+
+    public Car clone() throws CloneNotSupportedException {
```

Comment:
this method should be overriden

Reply:
You mean I must write "@overriding"? But in Idea already shown icon informing about overriding...

Reply:
https://beginnersbook.com/2014/07/override-annotation-in-java/

Reply:
Pavel, thank you for article. I understood, why I must write theese annotation.

Reply:
Please, review my file once more.